### ISSUE TITLE
Calling this.skip from Suite#before creates unexpected output

### ISSUE THREAD

**Issue by jason0x43** [theintern/intern#849]
In the report below, the 'tenon' suite was skipped by calling `this.skip` in a `before` method. The suite was reported as 'FAILED', and the suite summary indicates a suite error, but the tests were listed as passing. The overall summary, though, correctly indicates that 2 tests were not run, but incorrectly states that a suite error occurred.

```
Listening on localhost:9000 (ws 9001)
Tunnel started
Suite node - tenon FAILED
[object Object]
No unit test coverage for node
node: 2 passed, 0 failed; suite error occurred

‣ Created remote session chrome 62.0.3202.94 on Mac OS X (ad67b3d7f1e8e7739f3aeb797a5505e9)
✓ chrome 62.0.3202.94 on Mac OS X - aXe - external page (4.502s)
✓ chrome 62.0.3202.94 on Mac OS X - aXe - file name (0.328s)
No unit test coverage for chrome 62.0.3202.94 on Mac OS X
chrome 62.0.3202.94 on Mac OS X: 2 passed, 0 failed
TOTAL: tested 2 platforms, 2 passed, 0 failed, 2 not run; suite error occurred
```

Skipping a suite should not count as a suite error. It should report the suite itself as 'skipped', the suite summary should state that tests weren't run, and nothing should be flagged as an error or failure.

**Comment by jason0x43** [comment]
The above results are from the 'runner' reporter.

**Comment by jason0x43** [comment]
This is no longer an issue as of at least Intern 4.7. It was likely fixed by 73f5ca5.
